common/lib/xmodule/xmodule/js/spec/video/video_caption_spec.js

```diff
@@ -158,6 +171,54 @@
                 });
             });
 
+            describe('renderCaptions', function() {
+
+                describe('is rendered', function() {
+                    var KEY = $.ui.keyCode,
+
+                        keyPressEvent = function(key) {
+                            return $.Event('keydown', { keyCode: key });
+                        };
+
+                    it('toggle the captions on control click', function() {
+                        state = jasmine.initializePlayer();
```

Comment:
It looks like in the "on click" test the original state of the captions is _visible_, but in the "on keypress space/enter" tests the original state is _not visible_.   Why is that different?